Inspicio/wwwroot/js/multipleImages.js

```diff
@@ -10,30 +10,35 @@
         var description = $("#Image_Description_UserInput").val();
         var content = $("#b64uploader").val();
         if (title.length !== 0 && content.length !== 0) {
-
-            $("#uploader").css("width", "100%");
-            $(this).removeClass("ready");
-            var contentCol = listOfImages.map(v => v.Content);
-            if ($.inArray(content, contentCol) > -1) {
-             alert("This image is already added");
-             $("#uploader").html("").append(dropzoneMainText);
-             $(dropzoneMainText).show();
+            if (title.length < 5) {
+                alert("Screen title must have at least 5 characters!")
```

Comment:
When we had a single screen, we used the built in verification for length etc., but now since we have multiple screens, the form that wrapped all of the inputs was removed, so the verification does not work without it. The errors we encountered yesterday was because either the title of a screen or the project did not pass the verification in the controller. So basically this is just a placeholder for a proper verification method.